cli/uninstall/uninstall.go

```diff
@@ -226,25 +233,41 @@ func searchLatestVersion(linkName, binDst, headerDst string) (string, error) {
 		if util.Find(programsToSearch, programName) == -1 {
 			continue
 		}
+		isRightFormat, err := checkHashFormat(matches["ver"])
 
-		ver, err := version.Parse(matches["ver"])
 		if err != nil {
 			continue
 		}
-		if strings.Contains(programName, "tarantool") {
-			// Check for headers.
+
+		if isRightFormat && !strings.Contains(matches["ver"], ".") && !hashFound {
 			if _, err := os.Stat(filepath.Join(headerDst, binaryName)); os.IsNotExist(err) {
```

Comment:
Suppose, we should not skip, if there are no tt headers.

Reply:
fixed